### ISSUE TITLE
GWT SDM Code Server with Tomcat server

### ISSUE THREAD

**Issue by masterdany88** [gwt-plugins/gwt-eclipse-plugin#146]
Using eclipse plugin with maven projects and GWT SDM Code Server run and Tomcat8 Server cause often Tomcat server reload.

When I do a minor change in client side code, f.e.:  add or delete a blank line cause Tomcat server to reload, which is basically bad. Server Tomcat should only reload when Server side code changes.

When I used earlier version (google's official one) I did not have this problem. Only server side code change caused server restart. I use maven all the time, so I was running `mvn gwt:run` from console (not from ide, cause it wasn't supported).

**Comment by branflake2267** [comment]
Duplicate of https://github.com/gwt-plugins/gwt-eclipse-plugin/issues/140

**Comment by branflake2267** [comment]
There is a way to disconnect  the two servers from restarting. 

Goto the GWT Facet > GWT Facet settings 
![screen shot 2016-08-09 at 5 04 04 pm](https://cloud.githubusercontent.com/assets/1326504/17537672/7de62000-5e53-11e6-8048-351824c91852.png)

**Comment by branflake2267** [comment]
Closing duplicate

**Comment by branflake2267** [comment]
This disconnects the code server from restarting with the web server, or at least that's the intention. Did it work or not work for you?

**Comment by masterdany88** [comment]
As far I test it works. After unchecking, there is no server reload after client code change.
After changing server code I am getting immediately tomcat restart.

**Comment by branflake2267** [comment]
I need clarification when you say, "After changing server code...", do you mean that tomcat is restarting after a change? Is something going wrong?

**Comment by masterdany88** [comment]
"After changing server code I am getting immediately tomcat restart." is good behaviour. It is what I need. 
When I work on server part of code, I wonna to eclipse/tomcat/jvm to hotswap changes/ reload server, that I could see immediately my changes in app.
When I work on client part of code I wonna to Super Dev Mode to update my changes, that I could see them in browser in app. This work, so it is good. 

The problem was, that when I changed client part of code, tomcat was restarting, which was basically wrong, cause Tomcat deals only with server part, so there was no need to restart. It slowed down my development flow. 
Now thanks to Your answer, where You advised me to change in Project configuration: `GWT Facet settings`, it works as supposed. I am changing client code, I am getting updated code thanks to SDM. When I change server code I am getting Tomcat restart and I can see changes immediately.
Now there is no redundant server reload after changing client.

I hope I explained it enough. If there is some kind of ambiguity just ask.
